res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1712,6 +1940,10 @@ PioneerDDJSX.samplerLedsPlay = function(value, group, control) {
         sampleDeck = "[Sampler" + (samplerIndex + 1) + "]",
         padNum = PioneerDDJSX.samplerGroups[sampleDeck];
 
+    if (!engine.getValue(sampleDeck, "duration")) {
```

Comment:
There is now a `track_loaded` Control that would make the code clearer here than treating `duration` like a boolean.